src/main/java/cs/ualberta/cmput402/tictactoe/board/Board.java

```diff
@@ -128,5 +133,7 @@ public Player getPlayerAtPos(int row, int col){
         return board[row][col];
     }
 
+    public int getMoveCount() {return moveCount;}
```

Comment:
You should make the coding style consistent with other functions such as getCurrentPlayer() and getWinner().